frontend/src/components/Navbar/Navbar.css

```diff
@@ -3,4 +3,17 @@
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
-}
\ No newline at end of file
+}
+
+.logout-button {
+  margin-right: 10px;
+}
+
+.logout-button,
+.logout-button input:hover {
+  cursor: pointer;
+}
+
+.logout-button input {
+  margin-left: 2px;
+}
```

Comment:
We're moving away from dedicated css files and to using tailwind. I know the file still exists but will be removed eventually once we get around to removing them so I'd advise against adding to them (especially if we plan to remove them.) 

Tailwind can be used in conjunction with vanilla CSS but unless absolutely necessary (this will probably be necessary in the future), I'd suggest just using Tailwind.

Reply:
Ok removed css and used tailwind for this

Reply:
LGTM